packages/core/src/views/account-center/Micro.svelte

```diff
@@ -0,0 +1,100 @@
+<script lang="ts">
+  import { fade } from 'svelte/transition'
+  import { internalState$, wallets$ } from '../../streams'
+  import {
+    getDefaultChainStyles,
+    shortenAddress,
+    shortenEns,
+    unrecognizedChainStyle
+  } from '../../utils'
+  import { updateAccountCenter } from '../../store/actions'
+  import questionIcon from '../../icons/question'
+  import SuccessStatusIcon from '../shared/SuccessStatusIcon.svelte'
+  import WalletAppBadge from '../shared/WalletAppBadge.svelte'
+  import { state } from '../../store'
+
+  const { appMetadata } = internalState$.getValue()
+  const appIcon = (appMetadata && appMetadata.icon) || questionIcon
+  const chains = state.get().chains
+
+  $: [primaryWallet] = $wallets$
+  $: [firstAccount] = primaryWallet ? primaryWallet.accounts : []
+
+  $: [firstAddressAsset] =
+    firstAccount && firstAccount.balance
+      ? Object.keys(firstAccount.balance)
+      : []
+
+  $: primaryChain = primaryWallet && primaryWallet.chains[0]
+
+  $: validAppChain = chains.find(({ id, namespace }) =>
+    primaryChain
+      ? id === primaryChain.id && namespace === primaryChain.namespace
+      : false
+  )
+
+  $: defaultChainStyles = getDefaultChainStyles(primaryChain && primaryChain.id)
+
+  function maximize() {
+    updateAccountCenter({ expanded: true })
+  }
+</script>
+
+<style>
+  .minimized {
+    background-color: var(--onboard-white, var(--white));
+    border: 1px solid var(--onboard-gray-100, var(--gray-100));
+    width: 100%;
+    box-shadow: var(--onboard-shadow-3, var(--shadow-3));
+  }
+
+  .radius {
+    border-radius: 16px;
```

Comment:
I think we have a border radius variable we could use here

Reply:
Thanks for the suggestion. Added a new one for 16px.